lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -188,7 +186,7 @@ def decorated(*args, db: DatabaseSession, log_output: bool, fetch_batch_size: in
                         redcap_records[record.id].append(record)
 
             # Process all DETs in order of redcap_det_id
-            with pickled_cache(CACHE_FILE) as cache:
+            with pickled_cache(cache_file) as cache:
```

Comment:
What if cache_file == None here?

Reply:
If no cache file is provided, we use an in-memory cache.
The log will output:
```
[2020-10-28 09:40:01] No cache file provided; using transient, in-memory cache.
```